model/host/db.go

```diff
@@ -1254,14 +1255,18 @@ func FindDistroForHost(hostID string) (string, error) {
 	return h.Distro.Id, nil
 }
 
-func FindVolumesByUser(userID string) ([]Volume, error) {
+func FindVolumes(q bson.M) ([]Volume, error) {
 	volumes := []Volume{}
+	err := db.FindAll(VolumesCollection, q, db.NoProjection, db.NoSort, db.NoSkip, db.NoLimit, &volumes)
+
+	return volumes, err
+}
+
+func FindVolumesByUser(userID string) ([]Volume, error) {
```

Comment:
Should this be in host/volume? Or those queries should be in here, I'm not sure why they're split up to begin with

Reply:
moved both to volume.go